packages/ckeditor5-ui/tests/dropdown/utils.js

```diff
@@ -3,10 +3,10 @@
  * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
  */
 
-/* globals document, Event, console */
+/* globals document, Event, console, setTimeout */
 
 import { assertBinding } from '@ckeditor/ckeditor5-utils/tests/_utils/utils';
-import { keyCodes } from '@ckeditor/ckeditor5-utils/src/keyboard';
+import { global, keyCodes } from '@ckeditor/ckeditor5-utils';
```

Comment:
`document` is already ignored by the linter in line 6 and I don't see another usage of `global` here, so I wonder if it's needed? Also I'd be happy to clarify if there is a preferred approach for us in this matter as we are pretty inconsistent about it in the codebase. 

From the DX point of view I prefer using the `globals` clause as it saves some code, but API docs of `global` package suggest that in some environments (node?) such a solution won't work. Do you have some knowledge to drop?

Reply:
The point of the `global` module is having an elegant failsafe proxy to the `global/window` objects regardless of the environment ([see a recent PR](https://github.com/ckeditor/ckeditor5/pull/11999)). 

Technically speaking, there's nothing wrong with using the `/* globals */` declaration but given that every now and then we want to [stub the entire window object](https://github.com/ckeditor/ckeditor5/blob/bacb8cb5547d040ab3a4c175a78c8d6a98176eb3/packages/ckeditor5-ui/tests/panel/sticky/stickypanelview.js#L43) (because you cannot stub some of its properties, DOM is weird) I'd love to see the entire code-base use the `global` module just for consistency and ease of testing. 

If we introduce the editor to some new environment in the future, `global` would be the only point of failure (and the place to fix) instead of dozens of editing/UI features accessing `document` and `window` on their own.

Personally, I think we should forbid using `document` and `window` in `/* globals */` per ESLint configuration but this could be my OCD kicking in.